sdk/examples/xo_python/sawtooth_xo/xo_client.py

```diff
@@ -80,6 +82,28 @@ def show(self, name):
         except BaseException:
             return None
 
+    def _get_statuses(self, batch_ids):
+        try:
+            url = "http://{}/{}".format(self._base_url, 'batch_status')
+
+            if isinstance(batch_ids, bytes):
+                headers = {'Content-Type': 'application/octet-stream'}
+            else:
+                batch_ids = json.dumps(batch_ids).encode()
+                headers = {'Content-Type': 'application/json'}
+            headers['Content-Length'] = '%d' % len(batch_ids)
+
+            result = requests.post(url, data=batch_ids, headers=headers)
+            code, status = (result.status_code, result.json()["data"])
+
+            if code == 200 or code == 201 or code == 202:
```

Comment:
Even though you are sending it as a `POST`, any request to `/batch_status` is considered a `GET` in spirit, and will only return a `200` (or an error code).

_(A `POST /batches`, which actually submits something new, would return a `201` or `202`, but not a `200`)_